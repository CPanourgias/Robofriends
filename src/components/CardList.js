import React from "react";
import Card from "./Card";

const CardList = ({ robots }) => {
    return (
        <div>
            {
                robots.map(robot => {
                    return <Card
                        key = {robot.id}
                        username = {robot.username}
                        name = {robot.name}
                        email = {robot.email}
                        phone = {robot.phone}
                    />
                })
            }
        </div>
    );
};

export default CardList;