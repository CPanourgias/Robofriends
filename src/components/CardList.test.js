import { shallow } from "enzyme";
import React from 'react';
import CardList from './CardList';

it('Render Cardlist component', () => {
    const mockRobots = [
        {
            id: 1,
            name: 'John Snow',
            username: 'johnjohn',
            email: 'john@gmail.com'
        }
    ]
    expect(shallow(<CardList robots={mockRobots}/>)).toMatchSnapshot();
})