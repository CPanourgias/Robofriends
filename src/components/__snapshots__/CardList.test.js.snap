// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Cardlist component 1`] = `ShallowWrapper {}`;
